spec/index.html

```diff
@@ -1287,37 +1301,38 @@ <h2>Security Considerations</h2>
   <p>The RDF Abstract Syntax is not used directly for conveying information,
     although concrete serialization forms are specifically intended to do so.</p>
 
-  <p>Applications MAY evaluate given data to infer more assertions or to dereference IRIs,
+  <p>Applications MAY evaluate given data to infer more assertions or to dereference <a>IRIs</a>,
     invoking the security considerations of the scheme for that IRI.
     Note in particular, the privacy issues in [[RFC3023]] section 10 for HTTP IRIs.
     Data obtained from an inaccurate or malicious data source may lead to inaccurate or misleading conclusions,
     as well as the dereferencing of unintended IRIs.
-    Care must be taken to align the trust in consulted resources with the sensitivity of the intended use of the data;
-    inferences of potential medical treatments would likely require
-    different trust than inferences for trip planning.</p>
+    Care must be taken to align the trust in consulted resources with the sensitivity of
+    the intended use of the data;
+    inferences of potential medical treatments would likely require different trust than inferences
+    for trip planning.</p>
 
   <p>RDF is used to express arbitrary application data;
     security considerations will vary by domain of use.
     Security tools and protocols applicable to text
-    (e.g., PGP encryption, MD5 sum validation, password-protected compression)
-    may also be used on RDF documents.
+    (for example, PGP encryption, checksum validation, password-protected compression)
+    may also be used on N-Quads documents.
     Security/privacy protocols must be imposed which reflect the sensitivity of the embedded information.</p>
 
-  <p>RDF can express data which is presented to the user, for example, RDF Schema labels.
-    Applications rendering strings retrieved from untrusted RDF documents must ensure
-    that malignant strings may not be used to mislead the reader.
-    The security considerations in the media type registration for
-    XML ([[RFC3023]] section 10) provide additional guidance around the
-    expression of arbitrary data and markup.</p>
+  <p>RDF can express data which is presented to the user, such as RDF Schema labels.
+    Applications rendering strings retrieved from untrusted RDF documents,
+    or using unescaped characters,
+    SHOULD prevent such strings from being used to mislead the reader,
```

Comment:
The wording in the other instance of this warning is better. "prevent ... from being used to mislead" remains impossible, and SHOULD is no better than MUST for an impossibility.

Reply:
How about the following:

```suggestion
    SHOULD ensure that malignant strings may not be used to mislead the reader.
```

`SHOULD` seems better than `must`, even though the later is rendered to be non-normative. But, it seems that as this is a normative section, statements such as this should use normative wording.